doc/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting
+
+This documentation will help you to troubleshoot docker related issues while running your ccl unit test with "container" environment.
```

Comment:
The quotes around container seem unnecessary and the guidance is specific to a "docker container". Consider changing to `This section will help you to troubleshoot issues specific to running CCL Unit tests within a docker container.`

Reply:
Okay. I'll update it.